specification/metrics/amortizedcost.md

```diff
@@ -0,0 +1,42 @@
+# Amortized Cost
+
+Amortized cost represents the sum of the amortization of upfront fees, the amortization of recurring fees, and the remaining usage cost of the line item. This column is often used to show spending trends that include on demand rates, effective commitment-based discount rates, recurring commitments, as well as amortization of upfront fees. The fee and upfront amortization portion of this column’s value should be proportional to the number of units for the line item and the time granularity of the data. The currency for the value specified in Billed Cost can be found in the [Billing Currency](#billingcurrency) column. The aggregated Amortized Cost for a billing period MAY NOT match the charge received on the invoice for the same billing period.
+
+Practitioners are faced with two main challenges in respect to this metric:
+1. Practitioners need to amortize upfront fees over the duration of the commitment and distribute those fees to the appropriate reporting groups (e.g. tags, resources).
+2. Many commitment based discount constructs include a recurring expense for the commitment for every billing period and must distribute this cost to the resources using the commitment. This forces reconciliation between the initial commitment line item per period and the actual usage line items.
+
+The AmortizedCost column MUST be present in the billing data. This column MUST be a numeric value of type Decimal and MUST NOT contain null values. 
+
+## Column ID
+
+AmortizedCost
+
+## Display name
+
+Amortized Cost
+
+## Description
+
+The cost inclusive of amortized upfront charges, recurring fees, and remaining usage cost of the line item.
+
+### Concerning Granularity and Distribution of Recurring Fee
+
+While some providers may include a line item at the beginning of each applicable period for the amount of the commitment, Amortized Cost should include that cost distributed such that there is not a need to distribute this fee. Thus the practitioner is not required to distribute this commitment, but rather the provider must distribute these fees over the line items for the usage of this commitment. See examples section for further clarity.
```

Comment:
```suggestion
Some providers may include a line item at the beginning of a period for the applicable amount of a purchased commitment. Providers should distribute that cost within the Amortized Cost metric such that there is not a need to for users to distribute this fee. 
```

Reply:
Should we be more direct?
```suggestion
Providers should distribute the commitment purchase amount instead of including a line item at the beginning of a period so practitioners do not need to manually distribute the fee themselves.
```